### PR TITLE
bahar/update_dbot_banner_for_binary_bot

### PR DESCRIPTION
**Pull request by Bahar-FS**
Update DBot banner for Binary Bot

### CODE REVIEW

File: src/common/lang.js [hunk 1]
```diff
@@ -50,3 +50,16 @@ export const load = () => {
 
     addUiLang();
 };
+
+export const showRelatedBanner = () => {
```
Comment (salar-fs) [inline src/common/lang.js:54]: Could we name this better? What does `related banner` mean?

Comment (Bahar-FS) [reply src/common/lang.js:54]: Since the method is in the 'lang.js` I believe it's obvious that what does that mean, don't you think so?

Comment (salar-fs) [reply src/common/lang.js:54]: So why not just use `showBanner`? What happens if we're going to have other banners here as well?
IMO it's not descriptive enough.

File: src/botPage/view/View.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ import {
 import { translate } from '../../common/i18n';
 import { isEuCountry, showHideEuElements, hasEuAccount } from '../../common/footer-checks';
 import googleDrive from '../../common/integrations/GoogleDrive';
-import { getLanguage, showRelatedBanner } from '../../common/lang';
+import { getLanguage, showdBanner } from '../../common/lang';
```
Comment (carol-binary) [inline src/botPage/view/View.js:33]: Typo? 
```suggestion
import { getLanguage, showBanner } from '../../common/lang';
```

File: src/botPage/view/View.js [hunk 3]
```diff
@@ -188,7 +188,7 @@ const updateTokenList = () => {
 
         // If logged out, determine EU based on IP.
         isEuCountry(api).then(isEu => showHideEuElements(isEu));
-        showRelatedBanner();
+        showdBanner();
```
Comment (carol-binary) [inline src/botPage/view/View.js:191]: Here as well.
```suggestion
        showBanner();
```

File: src/botPage/view/View.js [hunk 4]
```diff
@@ -203,7 +203,7 @@ const updateTokenList = () => {
 
         const activeToken = getActiveToken(tokenList, getStorage(AppConstants.STORAGE_ACTIVE_TOKEN));
         showHideEuElements(hasEuAccount(tokenList));
-        showRelatedBanner();
+        showdBanner();
```
Comment (carol-binary) [inline src/botPage/view/View.js:206]: Here as well.
```suggestion
        showBanner();
```

File: src/common/lang.js [hunk 5]
```diff
@@ -51,7 +51,7 @@ export const load = () => {
     addUiLang();
 };
 
-export const showRelatedBanner = () => {
+export const showdBanner = () => {
```
Comment (carol-binary) [inline src/common/lang.js:54]:
```suggestion
export const showBanner = () => {
```

File: src/indexPage/index.js [hunk 6]
```diff
@@ -6,17 +6,16 @@ import Footer from './react-components/footer.jsx';
 import { oauthLogin } from '../common/appId';
 import { isEuCountry, showHideEuElements } from '../common/footer-checks';
 import GTM from '../common/gtm';
-import { load as loadLang , showRelatedBanner } from '../common/lang';
+import { load as loadLang, showdBanner } from '../common/lang';
```
Comment (carol-binary) [inline src/indexPage/index.js:9]:
```suggestion
import { load as loadLang, showBanner } from '../common/lang';
```

File: src/indexPage/index.js [hunk 7]
```diff
@@ -6,17 +6,16 @@ import Footer from './react-components/footer.jsx';
...
 import { createUrl } from '../common/utils/tools';
 import '../common/binary-ui/dropdown';
 
-
 const renderElements = () => {
     ReactDOM.render(<Logo />, document.getElementById('binary-logo'));
     ReactDOM.render(<Footer />, document.getElementById('footer'));
     isEuCountry().then(isEu => showHideEuElements(isEu));
-    showRelatedBanner();
+    showdBanner();
```
Comment (carol-binary) [inline src/indexPage/index.js:18]:
```suggestion
    showBanner();
```

**Review by carol-binary — APPROVED** [review]

**Pull request merged**